### PR TITLE
Improve Sign-In screen UI

### PR DESCRIPTION
**Pull request by sean-prz**
### Before  vs After
image
image

## How
- Removes create account button instead sign-in with google redirects you if it doesn't find your account
- Transformed the two green buttons into clickable text (with animation on alpha for feedback)
- Moved the buttons down to be closer to the thumb
## Why
Just to make the apps feel less like a student project

### CODE REVIEW

File: app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt [hunk 1]
```diff
@@ -12,34 +13,40 @@ import androidx.compose.foundation.layout.fillMaxWidth
...
-import com.github.se.cyrcle.ui.theme.ColorLevel
-import com.github.se.cyrcle.ui.theme.atoms.Button
 import com.github.se.cyrcle.ui.theme.atoms.GoogleSignInButton
 import com.github.se.cyrcle.ui.theme.atoms.Text
+import kotlinx.coroutines.launch
 
 @Composable
 fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewModel) {
   val context = LocalContext.current
+  val coroutineScope = rememberCoroutineScope()
```
Comment (LouisJamesVasseur) [inline app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt:49]: Why does the coroutineScope become necessary here?

**Review by LouisJamesVasseur — APPROVED** [review]
After testing everything manually, everything works as advertised. I see no issue with the code even after close inspection, the way you refactored makes sense and it looks good on the Screen. One thing I would say is maybe put the smaller underlined texts in italics? Let me know what you think. I like the way you re-linked the account creation to the Google sign-in, very ingenious! Good work!

**Comment by sean-prz** [comment]
> One thing I would say is maybe put the smaller underlined texts in italics? Let me know what you think. 

Yes I wanted to, but I did not manage to put the text in Italic so I did not waste more time on this, let me try again, I can copy another screen that has italic text

**Comment by sean-prz** [comment]
image
What do you think @LouisJamesVasseur

**Comment by LouisJamesVasseur** [comment]
> image
> What do you think @LouisJamesVasseur

Perfect! Make sure "Continue Offline" is also in italics for consistency!

File: app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt [hunk 2]
```diff
@@ -85,58 +94,90 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
                     when (it) {
                       UserViewModel.SignInFailureReason.ACCOUNT_NOT_FOUND -> {
                         Toast.makeText(context, accountNotFoundToast, Toast.LENGTH_LONG).show()
+                        navigationActions.navigateTo(Screen.CREATE_PROFILE)
```
Comment (Pijukai) [inline app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt:97]: As Louis said it is a clean way of refactoring the create account button into the google Sign in one, good and clean job !

File: app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt [hunk 3]
```diff
@@ -85,58 +94,90 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
...
+            val alphaOfOfflineButton = remember { Animatable(1f) }
+            suspend fun clickAnimation(alpha: Animatable<Float, AnimationVector1D>) {
+              alpha.animateTo(
+                  targetValue = 0.1f,
+                  animationSpec =
+                      tween(
+                          durationMillis = 100,
+                      ))
+            }
+            fun resetAlpha(alpha: Animatable<Float, AnimationVector1D>) {
```
Comment (Pijukai) [inline app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt:117]: If you are using this function only inside this class maybe you can put it as private ?

Comment (sean-prz) [reply app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt:117]: This function is already a local function, It can't be used outside of the SignInScreen. The compiler will not even let me write private

Comment (Pijukai) [reply app/src/main/java/com/github/se/cyrcle/ui/authentication/SignInScreen.kt:117]: Ok it is perfect then !

**Review by Pijukai — APPROVED** [review]
The rework works as expected so it is an APPROVED ! Also the style is really great so well done !
